markdown/blog/asyncapi-bounty-program-2024.md

```diff
@@ -0,0 +1,80 @@
+
+- 32 GitHub issues were resolved, [16 of which](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=-no%3Alevel+-round%3A%22bounty%2F2023-pilot%22&sliceBy%5BcolumnId%5D=77803059&sliceBy%5Bvalue%5D=advanced) had complexity level `Advanced`;
+
+- [50+ PRs](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5BcolumnId%5D=78904519&filterQuery=no%3Aresponsible) were merged in total;
+
+- [8000+ USD](https://opencollective.com/asyncapi/expenses?limit=30&tag=bounty&period=2023-09-30T21%3A00%3A00.000Z%E2%86%92all&collectiveSlug=asyncapi&status=PAID) were paid out as rewards.
+
+### Benefits for FOSS Community
+- at least [one issue](https://github.com/springwolf/springwolf-core/issues/820) was closed in a third-party project due to a fixed bug in AsyncAPI's software.
+
+
+### Benefits for AsyncAPI GitHub Organization
+Thanks to the Bounty Program Participants AsyncAPI got:
+
+- [TSC work automation](https://github.com/orgs/asyncapi/projects/36/views/4?sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=community%231093&filterQuery=) ([Aayush Saini](https://github.com/AayushSaini101));
+
+- new monorepo setup in [generator](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=&sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=generator%231044) and [parser-js](https://github.com/orgs/asyncapi/projects/36/views/4?filterQuery=&sliceBy%5BcolumnId%5D=78904519&sliceBy%5Bvalue%5D=parser-js%23963) ([Ayush Nautiyal](https://github.com/ayushnau));
+
+- saving of resources on DigitalOcean ([Ashish Padhy](https://github.com/shurtu-gal)); **(need a link)**
+
+- GitHub Action refactored from scratch to use CLI ([Ashish Padhy](https://github.com/shurtu-gal)); **(need a link)**
+
+- not just any but AMAZING [cheat sheet poster](https://github.com/asyncapi/website/issues/425#issuecomment-2121214509) ([Aishat Muibudeen](https://github.com/Mayaleeeee)) which is now used as a printout at conferences where AsyncAPI is hosted, as well as a giveaway.
+
+
+### Benefits for AsyncAPI Maintainers
+
+The Bounty Program gives AsyncAPI Maintainers possibility:
+
+- to announce a financial reward for GitHub issues that are currently in demand by the maintained project or the AsyncAPI Initiative in general, on which AsyncAPI Maintainers can either work themselves or delegate those to contributors who have more time and/or required expertise;
+
+-
+
+-
+
+
+### Benefits for Bounty Program Participants
+Usual benefits of the Bounty Program for its Participants who just start their career journey are:
+
+- development of a better approach to the evaluation of issues (it's fun reading messages from two months ago saying, '1-2 weeks left');
+
+- first/differing working experience (fully asynchronous distributed teams are still rare, unlike partially asynchronous distributed ones);
+
+- development of the possibility to showcase to the potential employer real-world work not obscured by an NDA.
+
+
+### Success Stories
```

Comment:
aren't benefits the same? or you mean some personal success stories? but I don't think that this was the goal of bounty program?

Reply:
I wrote this section when I was writing the skeleton of the article. It didn't come in handy, so removed.